include/meta/meta_fwd.hpp

```diff
@@ -35,8 +37,12 @@ namespace meta
 {
     inline namespace v1
     {
+#ifdef __cpp_lib_integer_sequence
+        using std::integer_sequence;
+#else
         template <typename T, T...>
         struct integer_sequence;
+#endif
```

Comment:
Why are we special-casing for libc++ here? Wouldn't we need to do the same thing for libstdc++?